[vtk] build failure

### Operating system

Windows 11 

### Compiler

MSVC vC143, CUDA 12.4

### Steps to reproduce the behavior

```Shell
`vcpkg install vtk[atlmfc,core,cuda,gdal,geojson,mpi,opengl,openvr,qt,sql,utf8,vtkm]`
```


### Failure logs

Building vtk[atlmfc,core,cuda,gdal,geojson,mpi,opengl,openvr,qt,sql,utf8,vtkm]:x64-windows@9.3.0-pv5.12.1#4...
-- Using cached Kitware-VTK-09a76bc55b37caad94d0d8ebe865caaed1b438af.tar.gz.
-- Extracting source C:/vcpkg/downloads/Kitware-VTK-09a76bc55b37caad94d0d8ebe865caaed1b438af.tar.gz
-- Applying patch FindLZMA.patch
-- Applying patch FindLZ4.patch
-- Applying patch libproj.patch
-- Applying patch pegtl.patch
-- Applying patch pythonwrapper.patch
-- Applying patch NoUndefDebug.patch
-- Applying patch fix-using-hdf5.patch
-- Applying patch FindExpat.patch
-- Applying patch cgns.patch
-- Applying patch vtkm.patch
-- Applying patch afxdll.patch
-- Applying patch vtkioss.patch
-- Applying patch jsoncpp.patch
-- Applying patch iotr.patch
-- Applying patch fast-float.patch
-- Applying patch fix-exprtk.patch
-- Applying patch devendor_exodusII.patch
-- Applying patch remove-prefix-changes.patch
-- Applying patch hdf5helper.patch
-- Applying patch opencascade-7.8.0.patch
-- Applying patch no-libharu-for-ioexport.patch
-- Applying patch no-libproj-for-netcdf.patch
-- Using source at C:/vcpkg/buildtrees/vtk/src/aed1b438af-b5254e961c.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v
    Working Directory: C:/vcpkg/buildtrees/vtk/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\vtk\config-x64-windows-dbg-CMakeCache.txt.log
      C:\vcpkg\buildtrees\vtk\config-x64-windows-rel-CMakeCache.txt.log
      C:\vcpkg\buildtrees\vtk\config-x64-windows-dbg-CMakeConfigureLog.yaml.log
      C:\vcpkg\buildtrees\vtk\config-x64-windows-rel-CMakeConfigureLog.yaml.log
      C:\vcpkg\buildtrees\vtk\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  ports/vtk/portfile.cmake:259 (vcpkg_cmake_configure)
  scripts/ports.cmake:192 (include)


error: building vtk:x64-windows failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle vtk:x64-windows: 1.5 min
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+vtk
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[vtk]+Build+error+on+x64-windows&body=Copy+issue+body+from+C%3A%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

### Additional context

It seems to be something wrong with cmake file of netcdf or vtk .

[config-x64-windows-out.log](https://github.com/user-attachments/files/16648722/config-x64-windows-out.log)

--- a/ports/vtk/vcpkg.json
+++ b/ports/vtk/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "vtk",
   "version-semver": "9.3.0-pv5.12.1",
-  "port-version": 4,
+  "port-version": 5,
   "description": "Software system for 3D computer graphics, image processing, and visualization",
   "homepage": "https://github.com/Kitware/VTK",
   "license": "BSD-3-Clause",
@@ -184,6 +184,7 @@
           "name": "vtk",
           "default-features": false,
           "features": [
+            "seacas",
             "vtkm"
           ]
         },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -9346,7 +9346,7 @@
     },
     "vtk": {
       "baseline": "9.3.0-pv5.12.1",
-      "port-version": 4
+      "port-version": 5
     },
     "vtk-dicom": {
       "baseline": "0.8.14",

--- a/versions/v-/vtk.json
+++ b/versions/v-/vtk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "99581eb538d39550007d25865540db2a368e3edb",
+      "version-semver": "9.3.0-pv5.12.1",
+      "port-version": 5
+    },
+    {
       "git-tree": "f66a99e8c041f8e084d2871c3cc1d48146ca90f0",
       "version-semver": "9.3.0-pv5.12.1",
       "port-version": 4

